### ISSUE TITLE
Undefined array key 1

### ISSUE THREAD

**Issue by cgdurha2** [fortrabbit/craft-copy#167]
I'm unable to successfully push anything up to my fortrabbit app as of today. Instead, I just get this error:

```
[ERROR] Undefined array key 1                                                                     
                                                                                                   
         yii\base\ErrorException                                                                   
                                                                                                   
         in /var/www/html/vendor/fortrabbit/craft-copy/src/Services/Git/GitonomyClient.php: 76
```

**Comment by pheeque1** [comment]
Hi @cgdurha2,

We recently replaced the git library in Craft Copy and this looks like an edge case from your particular setup. Please create a support ticket from the fortrabbit Dashboard so we can look into this further.

Thanks.

**Comment by cgdurha2** [comment]
Ok will do. For extra context, in case it helps anyone else, it was the first time I attempted to push that project up, and a git repo hadn’t been created yet. I downgraded to 2.3.2 and when I used copy/all/up it asked if I wanted to generate the repo and then finished everything else too.On Apr 18, 2023, at 1:45 AM, Rafik Abdulwahab ***@***.***> wrote:﻿
Hi @cgdurha2,
We recently replaced the git library in Craft Copy and this looks like an edge case from your particular setup. Please create a support ticket from the fortrabbit Dashboard so we can look into this further.
Thanks.

**Comment by pheeque1** [comment]
Thanks for providing the additional context. I was able to reproduce the bug with a project with no git repo. The [fix](https://github.com/fortrabbit/craft-copy/pull/168) is in review at the moment.

**Comment by pheeque1** [comment]
Thanks once again for filing the bug report, @cgdurha2 . The fix is now in [v2.4.1](https://github.com/fortrabbit/craft-copy/releases/tag/2.4.1).

**Comment by cgdurha2** [comment]
Thanks so much @pheeque1!
